### PR TITLE
Implement homework 1

### PR DESCRIPTION
**Pull request by ProffesorRabbit**

### CODE REVIEW

**Comment by extsoft** [comment]
@Tempos please review first

File: homeworks/nazar_kostiv/pyats01_hw.py [hunk 1]
```diff
@@ -3,46 +3,34 @@
...
-    }
 
 class SmokeTest(Testcase):
-    parameters = {
-        'num1': 3,
-        'num2': 0
-    }
 
     @test
     def test_add(self, steps, num1, num2):
```
Comment (Tempos) [inline homeworks/nazar_kostiv/pyats01_hw.py]: "self" is not used, so is not required
```suggestion
    def test_add(steps, num1, num2):
```

File: homeworks/nazar_kostiv/pyats01_hw.py [hunk 2]
```diff
@@ -3,46 +3,34 @@
...
-            except AssertionError:
-                self.skipped('Result is less then 0)')
-
-
-
+            if add(num1, num2) < 0:
+                step.skipped('Result is less then 0)')
 
     @test
     def test_devide(self, steps, num1, num2):
```
Comment (Tempos) [inline homeworks/nazar_kostiv/pyats01_hw.py]: "self" is not used, so is not required
```suggestion
    def test_devide(steps, num1, num2):
```

File: homeworks/nazar_kostiv/pyats01_hw.py [hunk 3]
```diff
@@ -3,46 +3,34 @@
 from pyats.aetest import Testcase, test, main
 from calculation import add, divide
```
Comment (Tempos) [inline homeworks/nazar_kostiv/pyats01_hw.py]:
```suggestion
# pylint: disable=C0103

```

Comment (ProffesorRabbit) [reply homeworks/nazar_kostiv/pyats01_hw.py]: @Tempos fixed

**Comment by ProffesorRabbit** [comment]
@Tempos Please review again

**Comment by ProffesorRabbit** [comment]
@extsoft Fixed merge conflict, please review once more time

**Comment by extsoft** [comment]
@ProffesorRabbit good job.

**Pull request merged**